src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -51,6 +52,10 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | CSKY         | `freg`         | `f[0-31]`                          | `f`                  |
 | s390x        | `reg`          | `r[0-10]`, `r[12-14]`              | `r`                  |
 | s390x        | `freg`         | `f[0-15]`                          | `f`                  |
+| Arm64EC      | `reg`          | `x[0-30]`                          | `r`                  |
+| Arm64EC      | `vreg`         | `v[0-15]`                          | `w`                  |
+| Arm64EC      | `vreg_low16`   | `v[0-15]`                          | `x`                  |
+| Arm64EC      | `preg`         | `p[0-15]`, `ffr`                   | Only clobbers        |
```

Comment:
Is SVE even allowed with arm64ec?

Reply:
I didn't see these registers being explicitly called out as not supported, but it looks like that's because none of the Qualcomm chips supported for Windows have SVE so it's not supported in general.